AI/Nullkiller/Pathfinding/Actors.cpp

```diff
@@ -70,7 +70,7 @@ int ChainActor::maxMovePoints(CGPathNode::ELayer layer)
 {
 #if AI_TRACE_LEVEL > 0
 	if(!hero)
-		throw std::exception("Asking movement points for static actor");
+		throw std::logic_error("Asking movement points for static actor");
```

Comment:
std::exception doesn't have non-default constructor. It's bug in CL implementation